wolox-books/src/app/app.component.scss

```diff
@@ -0,0 +1,60 @@
+@import '../sass/colors';
+@import '../sass/fonts';
+
+.navbar{
+  border-top: 5px solid $blue;
+  padding: 40px;
+  text-align: center;
+
+  .logo {
+    max-width: 500px;
+    display: block;
+    margin: 0 auto;
+  }
+
+  .subtitle{
+    font-weight: bold;
+    letter-spacing: 8px;
+    font-size: 1em;
+    margin-top: 10px;
+    max-width: 500px;
+  }
+}
+
+.main {
+  padding: 20px;
+  max-width: 500px;
+  margin: 0 auto;
+}
+
+.reg-form {
+
+  .prefix {
+    padding: 5px 7px;
+  }
+
+  .data {
+    background: $white;
+    border-radius: 8px;
+    width: 100%;
+    height: 35px;
+    margin-bottom: 15px;
+    padding: 10px;
+  }
+}
+
+.register {
+  background: $green;
+  color: #fff;
+  margin-top: 15px;
+}
+
+.separator {
+  margin: 20px 0;
+  border-top: 1px solid $green;
+}
+
+.login {
+  border: 1px solid $green;
```

Comment:
This button will be reused so I would recommend you separate this styles in a generic file